helm-charts/common/llm-ctrl-uservice/values.yaml

```diff
@@ -0,0 +1,104 @@
+# Copyright (C) 2024 Intel Corporation
+# SPDX-License-Identifier: Apache-2.0
+
+# Default values for llm-ctrl-uservice.
+# This is a YAML-formatted file.
+# Declare variables to be passed into your templates.
+
+replicaCount: 1
+vLLM_ENDPOINT: ""
+HF_HUB_DISABLE_PROGRESS_BARS: 1
+HF_HUB_ENABLE_HF_TRANSFER: 0
+
+image:
+  repository: opea/llm-vllm
+  pullPolicy: IfNotPresent
```

Comment:
Drop the value, it breaks CI testing for `latest` tag (see https://github.com/opea-project/GenAIInfra/pull/587).